test/inductor/test_torchinductor.py

```diff
@@ -8367,16 +8367,14 @@ def run(x):
             lambda: run(torch.randn([8, 32], device=self.device))
         )
 
-        if self.device == GPU_TYPE:
+        if is_halide_backend(self.device):
+            self.assertEqual(fw_code.count("halide_helpers.rand"), 1)
```

Comment:
it occurs once instead of twice because for Halide inductor only generates one kernel for dropout: `halide_kernel_0`, which is called twice. With triton, inductor generates `triton_poi_fused_native_dropout_0` and `triton_poi_fused_native_dropout_1`, each of which is called once.